pytest_trio/_tests/test_hypothesis_interaction.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+from hypothesis import given, strategies as st
+
+
+@given(st.integers())
+@pytest.mark.trio
+async def test_mark_inner(n):
+    pass
```

Comment:
Wouldn't it be interesting to add a `isinstance(n, int)` check in the body just to be sure ?